### DISCUSSION TITLE
How to add a new language to https://crowdin.com/project/xdrip ?

### DISCUSSION THREAD

**Discussion by erkexzcx** [NightscoutFoundation/xDrip#2497] [General]
Hello. I'd like to translate this application on https://crowdin.com/project/xdrip to Lithuanian language. There is no such language between the existing language (translations) and I can't find a button/link/guidance to add a new language. How do I do it?

Can only project's maintainers add language in there? If so, please add. I would like to translate to Lithuanian language for a friend that hardly understands other languages. Thanks!

**Comment by Navid200** [comment]
If you log into crowdin, there is a forum you can access.
You can make any suggestions and requests related to translations there as well.

**Reply by erkexzcx** [reply]
> If you log into crowdin, there is a forum you can access.
> You can make any suggestions and requests related to translations there as well.

**Please... Please read my initial message. :( It's only a few sentences long...**

Let me repeat myself, but with more details:
1. _I can't find a button/link/guidance to add a new language._
2. _There is no such language between the existing language (translations)_
3. _How do I do it?_ (how do I add additional language to the project?)
4. According to [upstream documentation](https://support.crowdin.com/changing-target-languages/), only a maintainer can add a new language.

**Reply by erkexzcx** [reply]
Also seems that I am missing some kind of a role/member status in that project. There is no such button to create additional thread in [Discussions page](https://crowdin.com/project/xdrip/discussions).

**Reply by Navid200** [reply]
And I will not repeat myself.
Only two sentences.

**Reply by Navid200** [reply]
I want to help.  But, there is no guarantee what I suggest solves your problem.
That's the whole point of a forum.  You start a thread.  You get plenty of responses.  You go through them and hopefully, you will find something useful.
But, every post is not going to exactly solve your problem.

There are also people who may find a thread through search because they are looking for something similar but may be not exactly the same thing.  So, a post may not help you but it may help someone else reading the same thread later.

But, if you act like either I have to solve your problem or else you will make it personal, then, I'd rather stay out.
I'm now worried what you are going post if I state my opinion about what you need to do so that you can post in Crowdin.

**Reply by erkexzcx** [reply]
> That's the whole point of a forum. You start a thread. You get plenty of responses. You go through them and hopefully, you will find something useful.

And I replied:

> Also seems that I am missing some kind of a role/member status in that project. There is no such button to create additional thread in [Discussions page](https://crowdin.com/project/xdrip/discussions).

@jamorham could you please help me with this case? @Navid200 is either uses some kind of reply template or has difficulties understanding what users are saying...

**Reply by erkexzcx** [reply]
@jamorham could you please advise how the new language (Lithuanian) in https://crowdin.com/project/xdrip is added? I am afraid only maintainers/administrators can add a new language.

**Reply by erkexzcx** [reply]
Even AI gives me better answers than you, @Navid200. I was asking you a very specific questions because I was genuinely trying to help this project, yet you decided to waste both of our time...

![image](https://user-images.githubusercontent.com/5955795/209338532-d517b8b9-3486-4a4e-bbee-c41c4568e5f7.png)

@jamorham ^^^

**Reply by jamorham** [reply]
I have enabled the Lithuanian language at crowdin, please let me know if this works.

**Reply by erkexzcx** [reply]
It is enabled now. Thanks. I've submitted a request to join the "language".
